modules/ossm-install-ossm-operator.adoc

```diff
@@ -8,10 +8,10 @@
 
 To install {ProductName}, install following Operators in this order. Repeat the procedure for each Operator.
 
-1. Optional: OpenShift Elasticsearch
-2. Jaeger
-3. Kiali
-4. {ProductName}
+* Optional: OpenShift Elasticsearch
+* Jaeger
+* Kiali
+* {ProductName}
 
```

Comment:
Can we move the Note to this file so that it's in the same module as the installation instructions?